lib/app.js

```diff
@@ -2,14 +2,22 @@ const ui = require('@asl/service/ui');
 const { crumbs } = require('@asl/service/ui/middleware');
 const { pages, assets, content, views } = require('@asl/pages');
 const {
-  establishments,
-  establishment,
-  details,
-  places,
-  projects,
-  people,
-  profile,
-  place
+  establishment: {
+    details,
+    dashboard,
+    list: establishments
+  },
+  place: {
+    list: places,
+    view: place
+  },
+  profile: {
+    list: people,
+    view: profile
+  },
+  project: {
+    list: projects
+  }
```

Comment:
I think this code would be clearer if it only destructured the top level properties, and then referenced the sub-properties when mounting.

i.e.

```
app.use('/details', establishment.details());
```

Reply:
right you are 👍

Reply:
Yeah, it's a bit funny to do a lot of work to add structure by grouping related concepts together, only to then immediately discard that structure.